Jenkinsfile

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env groovy
+
+// can't just use BUILD_TAG because qdr operator limits name of resources to 60 chars
+def namespace = env.JOB_BASE_NAME + '-' + env.BUILD_NUMBER
+namespace = namespace.toLowerCase()
+
+def stages_failed = false;
+
+def stf_resource = """
+apiVersion: infra.watch/v1beta1
+kind: ServiceTelemetry
+metadata:
+  name: default
+  namespace: ${namespace}
+spec:
+  alerting:
+    alertmanager:
+      storage:
+        strategy: ephemeral
+      receivers:
+        snmpTraps:
+          enabled: true
+  backends:
+    events:
+      elasticsearch:
+        enabled: true
+        storage:
+          strategy: ephemeral
+    metrics:
+      prometheus:
+        enabled: true
+        storage:
+          strategy: ephemeral
+  transports:
+    qdr:
+      enabled: true
+      deployment_size: 1
+      web:
+        enabled: false
+  elasticsearch_manifest: |
+    apiVersion: elasticsearch.k8s.elastic.co/v1beta1
+    kind: Elasticsearch
+    metadata:
+      name: elasticsearch
+      namespace: $namespace
+    spec:
+      version: 7.10.2
+      http:
+        tls:
+          certificate:
+            secretName: 'elasticsearch-es-cert'
+      nodeSets:
+        - config:
+            node.data: true
+            node.ingest: true
+            node.master: true
+            node.store.allow_mmap: true
+          count: 1
+          name: default
+          podTemplate:
+            metadata:
+              labels:
+                tuned.openshift.io/elasticsearch: elasticsearch
+            spec:
+              containers:
+                - name: elasticsearch
+                  resources:
+                    limits:
+                      cpu: '2'
+                      memory: 4Gi
+                    requests:
+                      cpu: '1'
+                      memory: 2Gi
```

Comment:
Is this change from 4G to 2G the only difference with your manifest vs. the original, or is there something else as well?

We really need a better way to make targeted changes like this.

Reply:
Yeah that's the only reason I don't deploy stf from the run-ci playbook. 2G consistently deploys on quicklab, 4G does not